docs/ref/forms/api.txt

```diff
@@ -830,107 +830,203 @@ The field-specific output honors the form object's ``auto_id`` setting::
+    associated form has an :attr:`~django.forms.Form.auto_id`. Returns an
+    empty string otherwise:
 
-    >>> data = {'subject': 'hi', 'message': '', 'sender': '', 'cc_myself': ''}
-    >>> f = ContactForm(data, auto_id=False)
-    >>> print(f['message'])
-    <input type="text" name="message" />
-    >>> f['message'].errors
-    ['This field is required.']
-    >>> print(f['message'].errors)
-    <ul class="errorlist"><li>This field is required.</li></ul>
-    >>> f['subject'].errors
-    []
-    >>> print(f['subject'].errors)
+    .. code-block:: html+django
 
-    >>> str(f['subject'].errors)
-    ''
+        <!-- wrap this field in a div with an unique ID -->
+        <div id="{{ field.auto_id }}">
+            <!-- don't actually render the field with a widget -->
+            <input type="hidden" name="{{ field.html_name }}" value="hidden value" />
+        </div>
 
-.. method:: BoundField.label_tag(contents=None, attrs=None, label_suffix=None)
+    .. note::
+       This attribute is calculated by the form only, so if the field's widget
+       overrides :meth:`~django.forms.Widget.id_for_label` this ID is likely to
+       be wrong.
 
-To separately render the label tag of a form field, you can call its
-``label_tag`` method::
+       If you use :meth:`~django.forms.BoundField.as_widget` or similar
+       methods that render the field with its widget (``__str__`` does this
+       too), use :attr:`~django.forms.BoundField.id_for_label` instead.
 
-    >>> f = ContactForm(data)
-    >>> print(f['message'].label_tag())
-    <label for="id_message">Message:</label>
+.. attribute:: BoundField.data
 
-Optionally, you can provide the ``contents`` parameter which will replace the
-auto-generated label tag. An optional ``attrs`` dictionary may contain
-additional attributes for the ``<label>`` tag.
+    This property returns the data for this :class:`~django.forms.BoundField`
+    extracted by the widget's :meth:`~django.forms.Widget.value_from_datadict`
+    method, or ``None`` if it wasn't given::
 
-The HTML that's generated includes the form's
-:attr:`~django.forms.Form.label_suffix` (a colon, by default) or, if set, the
-current field's :attr:`~django.forms.Field.label_suffix`. The optional
-``label_suffix`` parameter allows you to override any previously set
-suffix. For example, you can use an empty string to hide the label on selected
-fields. If you need to do this in a template, you could write a custom
-filter to allow passing parameters to ``label_tag``.
+        >>> unbound_form = ContactForm()
+        >>> print(unbound_form['subject'].data)
+        None
+        >>> bound_form = ContactForm(data={'subject': 'My Subject'})
+        >>> print(bound_form['subject'].data)
+        My Subject
 
-.. versionchanged:: 1.8
+.. attribute:: BoundField.errors
 
-    The label includes :attr:`~Form.required_css_class` if applicable.
+    A list-like object that is displayed as an HTML ``<ul class="errorlist">``
```

Comment:
I didn't find any documentation for `ErrorList`. The closest I could find was this:
https://docs.djangoproject.com/en/dev/ref/forms/api/#customizing-the-error-list-format
Should I link there?